service/service.go

```diff
@@ -89,6 +90,7 @@ func New(config Config) (*Service, error) {
 			SchemeBuilder: k8sclient.SchemeBuilder{
 				apiv1alpha2.AddToScheme,
 				infrastructurev1alpha2.AddToScheme,
+				providerv1alpha1.AddToScheme,
```

Comment:
For my understanding: Why do we need to add this?

Reply:
Good catch. We don't need it anymore.